src/test/java/seedu/address/model/person/predicates/RemindPredicateTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.model.person.predicates;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_COMPANY_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_POLICY_ISSUE_DATE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_POLICY_NO_AMY;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.PersonBuilder;
+
+public class RemindPredicateTest {
+
+    @Test
+    public void equals() {
+        int firstPredicateDay = 0;
+        int secondPredicateDay = 30;
+
+        RemindPredicate firstPredicate = new RemindPredicate(firstPredicateDay);
+        RemindPredicate secondPredicate = new RemindPredicate(secondPredicateDay);
+
+        // same object -> returns true
+        assertTrue(firstPredicate.equals(firstPredicate));
+
+        // same values -> returns true
+        RemindPredicate firstPredicateCopy = new RemindPredicate(firstPredicateDay);
+        assertTrue(firstPredicate.equals(firstPredicateCopy));
+
+        // different types -> returns false
+        assertFalse(firstPredicate.equals(1));
+
+        // null -> returns false
+        assertFalse(firstPredicate.equals(null));
+
+        // different person -> returns false
+        assertFalse(firstPredicate.equals(secondPredicate));
+    }
+
+    // All tests on the test method are done on the date 12/11/2023, and will be using 30 days as the benchmark.
+    // If the tests are going to be done after 12/12/2023, do adjust the expiry dates for testing purposes.
+    @Test
+    public void test_policyExpiryDateIsWithinUserInputDays_returnsTrue() {
```

Comment:
For a more sustainable test, maybe you could use a Date that is 30 days from now, so that the test doesn't have to constantly rewritten? Would help for CI too

Reply:
Will do, thanks for the recommendation!